backend/lcfs/db/migrations/versions/2024-11-28-01-09_b4da565bb711.py

```diff
@@ -0,0 +1,48 @@
+"""Add UCI Columns to FS and Export
+
+Revision ID: b4da565bb711
+Revises: 0775a141d335
+Create Date: 2024-11-28 01:09:21.241693
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from alembic_postgresql_enum import TableReference
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "b4da565bb711"
+down_revision = "0775a141d335"
```

Comment:
Is this correct down_revision?

Reply:
Good catch. Not at all, I made this MR before the squash. Updated to have the correct revision.